### DISCUSSION TITLE
SO Development Question

### DISCUSSION THREAD

**Discussion by AbdelrahmanAttia20** [Security-Onion-Solutions/securityonion#7346] [Unsupported Versions] [answered]
Hey guys, I want some help with how to change the design of the interface of security onion after installing and how to keep the same design for the next installation. in other words how to make my own distro/version of security onion, and how to customize it (keeping the same functionality.) thanks for your time I really appreciate your help.
My regards.

**Comment by dougburks** [comment, marked as answer]
https://docs.securityonion.net/en/2.3/soc-customization.html
